frontend/projects/shared/src/util/misc.util.ts

```diff
@@ -14,7 +14,7 @@ export function capitalizeFirstLetter(string: string): string {
   return string.charAt(0).toUpperCase() + string.slice(1)
 }
 
-export const exists = (t: any) => {
+export function exists<T>(t: T | undefined): t is T {
```

Comment:
:speech_balloon:  I used to be a fan of the const lambda pattern, but after debugging the function guarantee's that the function is named during debug.

Reply:
What's the benefit of using lambda const?

Reply:
None that, except that it force you to create a function before you use it. And it is consistent with other things, only sometimes. It is better with function, so :+1: